5-Cookies/server/services/MongoDBSerivce.ts

```diff
@@ -0,0 +1,37 @@
+import { errorHandler } from "../utils/errorHandler";
+import { MongoDB } from "../utils/MongoDB";
+
+require("dotenv").config("karate-kid-kickstart/5-Cookies/server/.env");
+const mongoose = require("mongoose");
+
+export class MongoDBService extends MongoDB {
+  connection;
```

Comment:
please assign a type when declaring class members

Reply:
Actually I don't use this connection so I removed it.